runelite-client/src/main/java/net/runelite/client/game/UntradeableItemMapping.java

```diff
@@ -53,7 +53,20 @@
 	CRYSTAL_LEGS(ItemID.CRYSTAL_LEGS, 2, ItemID.CRYSTAL_ARMOUR_SEED),
 	CRYSTAL_LEGS_INACTIVE(ItemID.CRYSTAL_LEGS_INACTIVE, 2, ItemID.CRYSTAL_ARMOUR_SEED),
 	CRYSTAL_BODY(ItemID.CRYSTAL_BODY, 3, ItemID.CRYSTAL_ARMOUR_SEED),
-	CRYSTAL_BODY_INACTIVE(ItemID.CRYSTAL_BODY_INACTIVE, 3, ItemID.CRYSTAL_ARMOUR_SEED);
+	CRYSTAL_BODY_INACTIVE(ItemID.CRYSTAL_BODY_INACTIVE, 3, ItemID.CRYSTAL_ARMOUR_SEED),
+
+	// can be traded in at Logosia
+	TATTERED_MOON_PAGE(ItemID.TATTERED_MOON_PAGE, 1000, ItemID.COINS_995),
+	TATTERED_SUN_PAGE(ItemID.TATTERED_SUN_PAGE, 1000, ItemID.COINS_995),
+	TATTERED_TEMPLE_PAGE(ItemID.TATTERED_TEMPLE_PAGE, 1000, ItemID.COINS_995),
+
+	// can be traded in at Barlak
+	LONG_BONE(ItemID.LONG_BONE, 1000, ItemID.COINS_995),
+	CURVED_BONE(ItemID.CURVED_BONE, 1000, ItemID.COINS_995),
```

Comment:
The wiki states that the price is received is 2000 coins.

https://oldschool.runescape.wiki/w/Curved_bone